app/assets/stylesheets/home/components/psychologist_page/psychologist_page.scss

```diff
@@ -0,0 +1,109 @@
+@import '../variables';
+
+.psychologist-info{
+  align-items: center;
+  @include flex-column;
```

Comment:
it is a good practice to have the `@include` - `@entend` keywords at the beginnin of the properties

```
.psychologist-info {
  @include flex-column;
  align-items: center;
}
```

Reply:
Imma change it too!